src/apps/trade/TradeApp.tsx

```diff
@@ -0,0 +1,112 @@
+import React from 'react'
+import styled from 'styled-components'
+import { BrowserRouter, HashRouter, Route, Switch, Link } from 'react-router-dom'
+import { hot } from 'react-hot-loader/root'
+
+import { MEDIA } from 'const'
+
+import { withGlobalContext } from 'hooks/useGlobalState'
+import useNetworkCheck from 'hooks/useNetworkCheck'
+import Console from 'Console'
+import { GlobalModalInstance } from 'components/OuterModal'
+import { rootReducer, INITIAL_STATE } from 'reducers-actions'
+
+import { GenericLayout } from 'components/layout'
+import { Menu } from 'components/layout/GenericLayout/Menu'
+import { NavTools } from 'components/layout/GenericLayout/NavTools'
+
+import PortfolioImage from 'assets/img/portfolio.svg'
+import PortfolioImageWhite from 'assets/img/portfolio-white.svg'
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const Router: typeof BrowserRouter & typeof HashRouter = (window as any).IS_IPFS ? HashRouter : BrowserRouter
+
+const NotFound = React.lazy(
+  () =>
+    import(
+      /* webpackChunkName: "Extra_routes_chunk"*/
+      './pages/NotFound'
+    ),
+)
+
+const Trading = React.lazy(
+  () =>
+    import(
+      /* webpackChunkName: "Trade_chunk"*/
+      './pages/Trading'
+    ),
+)
+
+const PortfolioLink = styled.li`
+  margin: 0 2.4rem 0 0;
+
+  @media ${MEDIA.mediumDown} {
+    order: 2;
+  }
+
+  > a::before {
+    display: block;
+    margin: 0 0.8rem 0 0;
+    width: 1.6rem;
+    height: 1.4rem;
+    content: '';
+    background: url(${PortfolioImage}) no-repeat center/contain;
+  }
+
+  &:hover > a::before {
+    background-image: url(${PortfolioImageWhite});
+  }
+`
+
+export const SwapAppV1: React.FC = () => {
```

Comment:
should be renamed to `TradeApp` or something

Reply:
also likely doesn't need to be exported?

Reply:
I kind of like the names exports, so the component decide on the name of the exported item (and also support multiple items to export). But I exported as default too, since the component it makes sense to be the default. I have no strong opinion, so I think I'll remove then just the default

Reply:
I'll do it in the tip of the waterfall

Reply:
> should be renamed to TradeApp or something

I think the old web is more of a swap UI than trade ui, I think. Whatever we call it, it should have the V1 since it's not migrated for protocol 2

Reply:
Yeah i'm fine with the named export :+1:

Reply:
I disagree with swap v1.
Even though kind of there's a swap element, there's also trade mixed in, together with liquidity.
I look at swap v1 and think it's the first version of the swap app (the swap MVP).

I'd rather call it gpV1Ui == Gnosis Protocol V1 User Interface. We could also use the long name to be very explicit